src/main.rs

```diff
@@ -28,6 +29,7 @@ static DISALLOWED_SHORTHAND_PROJECT_NAMES: [&str; 4] = ["new", "edit", "load", "
 
 static USAGE: &str = "
 Usage:
+    muxed (list | ls)
```

Comment:
I kinda like being able to do `list` or `ls` but not sure what you're thinking in this regards. Docopt doesn't seem to make this trivial, but this is working

I also had to put this first, or else docopt would parse it as the 'shorthand' load

Reply:
I'm pretty much right with you. I like the `ls` shorthand, it's annoying to have to make a whole extra cmd for it but that's also not a big enough blocker to _not_ do it. So I'm happy to keep it.

I played around so much this weekend trying to grasp why `list` can't be lower without getting parsed wrong. I can't figure it out! Also not a big enough problem to not do it. So it'll stay at the top for now, that's totally fine.

Reply:
> I played around so much this weekend trying to grasp why list can't be lower without getting parsed wrong. I can't figure it out! 

I _think_ I wrapped my head around this while working, will try to get an explanation written up if I can recall the context 😆

Reply:
I finally get it. It's pattern matching from the top down in the list.
`muxed [flags] [options] <project>` 
and
`muxed (list | ls)`
Are the same thing as far as a pattern match is concerned. `[flags] [options]` are optional and `<project>` is any argument.

so `muxed list` meets the first pattern where `list` becomes the `<project>` argument. Putting `muxed (list | ls)` first allows us to match it before `list` falls into the ambiguous argument.